.npmrc

```diff
@@ -1,3 +1,17 @@
+# Override users global npmrc
+registry = https://registry.npmjs.org/
+@sap-ux:registry = https://registry.npmjs.org/
+@sap:registry = https://registry.npmjs.org/
+@sapux:registry = https://registry.npmjs.org/
+@types:registry = https://registry.npmjs.org/
+
+audit = true
+package-lock = true
```

Comment:
Not used because you do not use npm to install.

Reply:
pnpm seems to respect this as I had it set to false initially and the lock file was not updated and getting a warninn in the console